app/config.js

```diff
@@ -8,17 +8,22 @@ const {cfgPath, cfgDir} = require('./config/paths');
 const watchers = [];
 let cfg = {};
 let _watcher;
+let _isWatching = false;
 
 const _watch = function() {
   if (_watcher) {
     return _watcher;
   }
 
   const onChange = () => {
-    cfg = _import();
-    notify('Configuration updated', 'Hyper configuration reloaded!');
-    watchers.forEach(fn => fn());
-    checkDeprecatedConfig();
+    if (_isWatching == false) {
```

Comment:
You should use `===` 

But more important, I think that `_isWatching` will ever be `true` because all next actions are synchronous.
Can you explain how it can be false ?